com.unity.render-pipelines.universal/Shaders/Particles/ParticlesSimpleLitInput.hlsl

```diff
@@ -30,6 +30,8 @@ TEXTURE2D(_SpecGlossMap);       SAMPLER(sampler_SpecGlossMap);
 #define CAMERA_NEAR_FADE _CameraFadeParams.x
 #define CAMERA_INV_FADE_DISTANCE _CameraFadeParams.y
 
+#define _BumpScale 1.0
```

Comment:
i suspect this could be implemented properly, to make particle shaders more consistent with other URP shaders.
Not that i think you should feature-creep this into this PR - just something to consider for a future PR ;-)

Reply:
Yep exactly. I will note this down and place as a future task :)